OSDOCS-20033: Kueue 1.4 and DRA (#113996)
openshift/openshift-docs: modules/kueue-release-notes-1.4.adoc

  - Tracks usage over time with a configurable decay function
  - Applies immediate admission penalties to prevent resource monopolization

For more information, see xref:../../ai_workloads/kueue/admission-fair-sharing.adoc#admission-fair-sharing[Admission fair sharing].

Dynamic Resource Allocation (DRA) quota management for GPUs (Technology Preview)::

> {kueue-name} now supports quota management for workloads that request GPUs through Dynamic Resource Allocation (DRA). When configured, {kueue-name} tracks DRA device requests toward quota alongside traditional resources such as CPU and memory, preventing teams from exceeding their allocated GPU resources.

Reviewer: Be careful using the word "supports" when you're referring to a tech preview feature.

```suggestion
You can manage quotas for workloads that request GPUs through Dynamic Resource Allocation (DRA). When quota management is configured, {kueue-name} tracks DRA device requests toward quota alongside traditional resources such as CPU and memory, preventing teams from exceeding their allocated GPU resources.
```
